### ISSUE TITLE
LICENSE issue

### ISSUE THREAD

**Issue by Eutropios** [urllib3/urllib3#3035]
Contributors.txt?

In your LICENSE file, it's stated that the project is licensed by the owner and contributors, and then point to a file named CONTRIBUTORS, but such a file doesn't exist. I also search for an AUTHORS file but such a file doesn't exist either.

**Comment by shazow** [comment]
We've had so many contributors over the decades that it became impractical to maintain a static file in the repository, especially when Github does a pretty good job: https://github.com/urllib3/urllib3/graphs/contributors

Worth updating the LICENSE file to remove the old `contributors.txt` reference?

**Comment by sigmavirus24** [comment]
I don't think so. Most contributors contribute under the expectation they're giving up copyright so it's not necessary to attribute copyright to the whole list. Further that makes a legal mess for the future

**Comment by shazow** [comment]
IANAL etc but my understanding is that by default each contributor retains their own copyright, that's why many projects have a copyright assignment process before accepting PRs, and that's why it's difficult to change the license of a project without an explicit consolidated copyright owner (because you need each contributors' permission or remove their contributions).

That is to say, my intention when I wrote that line was to stay within the default spirit of the license in the first place. I don't think removing the outdated file reference would make any legal difference or intent difference or expectation difference.

If we really want to avoid touching the LICENSE.txt file, could just add back the contributors.txt file that only contains a link to the Github contributors. :)

**Comment by sethmlarson** [comment]
I'm in agreement that we remove the outdated reference.
